internal/platform/cpuid_arm64.go

```diff
@@ -0,0 +1,76 @@
+//go:build gc
```

Comment:
why is this file suffixed with arm64 but it contains logic?

Reply:
Because I'm dumb. Thanks for noticing!